Core/GDCore/IDE/Events/ExpressionValidator.h

```diff
@@ -293,12 +293,14 @@ class GD_CORE_API ExpressionValidator : public ExpressionParser2NodeWorker {
     forbidsUsageOfBracketsBecauseParentIsObject = false;
 
     Type currentParentType = parentType;
+    Type currentChildType = childType;
```

Comment:
Should you also save and restore `forbidsUsageOfBracketsBecauseParentIsObject`?

Reply:
Well, it's actually reseted just before. Is there a risk the `Visit` is setting it back to `true` (meaning we should again reset it to false just after)?

Reply:
I think it won't be a problem. It's only a message between a `VariableNode` and its `VariableBracketAccessorNode` if any.